README.rst

```diff
@@ -136,16 +136,20 @@ How to write django-admin2 modules
   djadmin2.default.register(Comment)
   djadmin2.default.register(User, UserAdmin2)
 
-Migrating from 0.5.x
+Migrating from 0.6.x
```

Comment:
I suggest to create new section for migration 0.5.x -> 0.6.x . I think it's better than overwriting old migration path.